### PR TITLE
[add]delete metod tests.

### PR DESCRIPTION
**Pull request by AndrzejczakSebastian**

### CODE REVIEW

File: spec/controllers/items_controller.rb [hunk 1]
```diff
@@ -62,4 +62,24 @@
...
+      expect(subject).to redirect_to(root_path)
+    end
+
+    it 'should redirect with notice' do
+      subject
+      expect(flash[:notice]).to be_present
+    end
+
+    it 'should destroy author' do
+      item_id = item.id
```
Comment (Schocker) [inline spec/controllers/items_controller.rb]: What does that line do? I don't see item_id used anywhere.

Comment (AndrzejczakSebastian) [reply spec/controllers/items_controller.rb]: It's item id(paramater) for delete method. Without this line destroy action will not delete item.

Comment (Schocker) [reply spec/controllers/items_controller.rb]: But you're not passing this variable to destroy method. `id: item`, not `item_id`. That's why I asked.

Comment (AndrzejczakSebastian) [reply spec/controllers/items_controller.rb]: It's for creating item with let. I changed this lane to be more clear.

**Pull request merged**